.github/actions/create-venv-for-tests/action.yml

```diff
@@ -14,7 +14,7 @@ runs:
     # These tests are slow hence will only run on linux.
     # This env will be used to install ipykernel & test for prompts if ipykernel is missing & imilar tests.
     # Ensure this is registered as a kernel.
-    - name: Create virtual environment for Tests
+    - name: Create virtual environment without ipykernel
```

Comment:
Doesn't this install ipykernel?

Reply:
& then it uninstalls it (few lines below in the script).
We install it so that we can register the kernel.